src/query/storages/fuse/src/io/read/block_reader.rs

```diff
@@ -33,7 +40,11 @@ use common_storage::ColumnLeaf;
 use common_storage::ColumnLeaves;
 use common_storages_table_meta::meta::ColumnMeta;
 use futures::future::try_join_all;
+use futures::Stream;
+use futures_util::TryStreamExt;
+use opendal::raw::output::Read as DALRead;
```

Comment:
Can we use `futures::Stream<Item=io::Result<Bytes>>` directly? It's better not to depend on opendal's `raw` API.